reflex/components/component.py

```diff
@@ -1532,17 +1535,15 @@ def _get_added_hooks(self) -> dict[str, ImportDict]:
         code = {}
 
         def extract_var_hooks(hook: Var):
-            _imports = {}
             var_data = VarData.merge(hook._get_all_var_data())
             if var_data is not None:
                 for sub_hook in var_data.hooks:
-                    code[sub_hook] = {}
-                if var_data.imports:
-                    _imports = var_data.imports
+                    code[sub_hook] = None
```

Comment:
how does this pass pyright when the return value is annotated as `dict[str, VarData]`?

maybe this should be `dict[str, VarData | None]` and then on like 1555 you can set `code[hook] = None` as well

Reply:
Maybe because that line is inside an internal function `extract_var_hooks`, it might trick pyright ? not sure on that one.
Made the change anyway.